sdk/keyvault/keyvault-certificates/test/lro.create.test.ts

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as assert from "assert";
+import { CertificateClient, KeyVaultCertificate } from "../src";
+import { testPollerProperties } from "./utils/recorderUtils";
+import { env } from "@azure/test-utils-recorder";
+import { authenticate } from "./utils/testAuthentication";
+import TestClient from "./utils/testClient";
+import { PollerStoppedError } from "@azure/core-lro";
+
+describe("Certificates client - Long Running Operations - create", () => {
+  const certificatePrefix = `recover${env.CERTIFICATE_NAME || "CertificateName"}`;
+  let certificateSuffix: string;
+  let client: CertificateClient;
+  let testClient: TestClient;
+  let recorder: any;
+
+  const basicCertificatePolicy = {
+    issuerName: "Self",
```

Comment:
can you use the default policy at least once so we can ensure to test that too?

Reply:
For sure! I can use the default policy everywhere too. Should I do that instead?

Reply:
I'll do what you say for this PR, but the question remains! As in, for the future.